sympy/printing/octave.py

```diff
@@ -34,18 +34,21 @@
 # generally a mapping to (argument_conditions, octave_function).
 known_fcns_src2 = {
     "Abs": "abs",
+    "arg": "angle",
```

Comment:
Unfortunately, this is not identical to SymPy's function. In SymPy, arg(-1) is pi. In Octave, arg(-1) or angle(-1) is -pi. 

By the way, why not use `arg`? They are synonymous, the [Octave docs](https://www.gnu.org/software/octave/doc/v4.2.1/Complex-Arithmetic.html) give arg in the example, and it's more familiar to SymPy users.

Reply:
I should've left a comment: in Matlab, its called (only) `angle`.

At some point, we might want to split into Octave and Matlab dialects but for now I try for both.